### DISCUSSION TITLE
Agenda 2021-09-21

### DISCUSSION THREAD

**Discussion by afoxman** [microsoft/rnx-kit#628] [Bundle Working Group]
Welcome to the Bundle Working Group!

Introductions

Choosing a form of communication

Topics of Interest
* Developer pain points
* Metro remote caching
* esbuild
* Metro module ids

Open Discussion

Wrap-up

**Comment by majames** [comment]
Could we also have an agenda item for consolidating on a form of communication moving forward? (i.e. picking one or two of GH discussions, email, Slack, FB messenger, Discord etc... to use for comms)

**Reply by kelset** [reply]
Based on my experience so far, I think we should do:
* emails for "broad announcements" and stuff
* GH for agenda/meeting notes
* discord for more direct comms

**Comment by raejin** [comment]
I would like to chat about expanding Metro module id system to support string or other module id types.

We are exploring using hash module ids in our Metro build. The Metro internal code assumes module id to be a number, so we had to duplicate Metro serialization code and replacing the sort function.

We would like to get more thoughts on this topic from everyone, thanks!
